dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -283,7 +285,7 @@ protected boolean rewriteDockerfile(String img, String tag,
         boolean modified = false;
         while ( (line = reader.readLine()) != null ) {
             /* Once true, should stay true. */
-            modified = changeIfDockerfileBaseImageLine(img, tag, strB, line, ignoreImageString) || modified;
+            modified = changeIfDockerfileBaseImageLine(reader, img, tag, strB, line, ignoreImageString) || modified;
```

Comment:
I think it would be better to pass `reader.readLine()` into this call b/c it'll be a lot clear to unit test.

Reply:
I don't think you can with this implementation.  This change works based upon the fact that if the line is a from instruction you can read the next line in the stream to determine if its a comment.  This is making the parser a look ahead 1 parser with respect to lines instead of a look ahead 0 parser.

Reply:
Updated the solution to handle this.